### DISCUSSION TITLE
Connect to database using SQL Server Instance Name

### DISCUSSION THREAD

**Discussion by Alv-sys** [directus/directus#8773] [Archive]
Hi there!
I am having issues trying to connect with Directus by using a SQL Server Instance name (SERVER/INSTANCE).

By using it in the install process it doesn't work, I tried with all the combinations of symbols but didn't work, so I tried to find the answer in the dependency Tedious, finding [this](https://github.com/tediousjs/tedious/issues/20), which suggests using this configuration:

`var config = {
    userName: 'test',
    password: 'test',
    server: 'hostname',
    options: {
        instanceName: 'MYINSTANCENAME'
    }
};`

but I couldn't find the way to do it in Directus.
How could I use an instance name?

I would really appreciate any help

**Comment by azrikahar** [comment]
Hi! Directus environment variables does allow this flexibility of nesting configs using double underscores, as shown here: https://docs.directus.io/reference/environment-variables/#type-casting-and-nesting

Based on the same concept shown in the docs, you should be able to specify the `instanceName` inside `options` by doing `DB_OPTIONS__INSTANCE_NAME="MYINSTANCENAME"` (note the double underscore between OPTIONS and INSTANCE)

**Reply by Alv-sys** [reply]
Awesome!! 
Thank you very much
